documentation/cookbook/working-with-test-suite.markdown

```diff
@@ -80,11 +79,8 @@ Example:
 ```bash
 DB_HOSTNAME=192.168.0.3 DB_USER=nonroot DB_PW=topsecret ./tests/bin/setup.mysql.sh
 DB_HOSTNAME=192.168.0.3 DB_USER=postgres DB_PW=secret ./tests/bin/setup.pgsql.sh
-./tests/bin/setup.sqlite.sh #does not have any configuration ability
```

Comment:
maybe we should add here something so people know they have to remove the sqlite file to reset the test database.